spec/router_spec.js

```diff
@@ -142,6 +143,98 @@ describe("Router", function() {
     });
   });
 
+  describe("navigate", function() {
+    beforeEach(function() {
+      this.router = new Router({ $el: $('<div>') });
+    });
+
+    it("calls router.navigate with provided options", function() {
+      spyOn(Backbone.Router.prototype, 'navigate');
+
+      const view = new View();
+      view.render();
+      this.router.activeView = view;
+
+      this.router.navigate("nextPage", { trigger: false });
+      expect(Backbone.Router.prototype.navigate).toHaveBeenCalledWith("nextPage", { trigger: false });
+    });
+
+    it("calls router.navigate with options defaulted to true if none are passed", function() {
+      spyOn(Backbone.Router.prototype, 'navigate');
+
+      const view = new View();
+      view.render();
+      this.router.activeView = view;
+
+      this.router.navigate("nextPage");
+      expect(Backbone.Router.prototype.navigate).toHaveBeenCalledWith("nextPage", true);
+    });
+
+    describe("without beforeNavigate hook", function() {
+      describe("when activeView is provided", function() {
```

Comment:
I think the nesting here should be switched. The `beforeNavigate` hook doesn't mean anything without an `activeView`.

The nesting should be something like:
```
with active view:
  without beforeNavigate hook
  with beforeNavigate hook

without active view:
```